.github/workflows/docker.yml

```diff
@@ -21,12 +21,46 @@
 # SOFTWARE.
 ---
 name: docker
-'on':
+on:
   push:
   pull_request:
+
 jobs:
   docker:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - run: docker build . -t home
+      - uses: docker/setup-qemu-action@v2
+      - uses: docker/setup-buildx-action@v2
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            yegor256/squid-proxy
+          # yamllint disable rule:line-length
+          tags: |
+            type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', 'master') }}
+            type=ref,event=branch
+            type=ref,event=pr
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
```

Comment:
@iloveitaly I'm curious, where will these `version`, `major`, and `minor` come from?

Reply:
these are autogen'd from the PR. They aren't great, so we could remove if you want.

Reply:
@iloveitaly I don't know what would be a better solution... Maybe let's just increment the version and that's it? Every new push to master generates a new version in Docker Hub. Just a number, no major, no minor -- only an integer. Is it possible?